src/data_structures/fmindex.rs

```diff
@@ -168,18 +183,46 @@ impl FMDIndex {
     ///   less memory usage, but worse performance)
     pub fn new(bwt: BWT, k: usize) -> Self {
         let mut alphabet = dna::n_alphabet();
-        alphabet.insert(b'$');
-        assert!(
-            alphabet.is_word(&bwt),
-            "Expecting BWT over the DNA alphabet (including N) with the sentinel $."
-        );
+        alphabet.insert(SENT_CHAR);
+        assert!(alphabet.is_word(&bwt),
+                "Expecting BWT over the DNA alphabet (including N) with the sentinel $.");
 
         FMDIndex {
             fmindex: FMIndex::new(bwt, k, &alphabet),
-            revcomp: dna::RevComp::new()
+            revcomp: dna::RevComp::new(),
         }
     }
 
+    /// Find longest non-encompassing prefix match
+    pub fn longest_prefix_match(&self, pattern: &[u8], min_overlap: usize) -> Option<BiInterval> {
+        let exact_match_interval = self.fmindex.backward_search(pattern.iter());
```

Comment:
I have fixed them in the master branch.
On Wed, 2016-03-02 at 12:28 -0800, Taylor C. wrote:
> In src/data_structures/fmindex.rs:
> >          }
> >      }
> >  
> > +    /// Find longest non-encompassing prefix match
> > +    pub fn longest_prefix_match(&self, pattern: &[u8],
> min_overlap: usize) -> Option {
> > +        let exact_match_interval =
> self.fmindex.backward_search(pattern.iter());
> Ah, so those were supposed to be pub... That makes much more sense
> now. :)
> —
> Reply to this email directly or view it on GitHub.